src/meta/src/stream/stream_graph/fragment.rs

```diff
@@ -620,8 +624,11 @@ impl CompleteStreamFragmentGraph {
 
                             (source_job_id, edge)
                         }
-                        _ => {
-                            // handle other kinds of streaming graph, normally MV on MV
+                        DdlType::MaterializedView | DdlType::Sink | DdlType::Index => {
+                            // handle MV on MV
+
+                            // Build the extra edges between the upstream `Materialize` and the downstream `StreamScan`
+                            // of the new materialized view.
                             let mview_fragment = upstream_root_fragments
```

Comment:
I think this can also be `source_fragment` for MV on backfill-able source later.

Reply:
Wait, isn't this also the same for CDC table..?

Reply:
Oh, the only difference is how to get `output_indices`, we have `as_materialize` in this branch.